spring-cloud-deployer-cloudfoundry/src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/ApplicationLogAccessor.java

```diff
@@ -54,29 +56,32 @@ public ApplicationLogAccessor(LogCacheClient logCacheClient) {
      * @return String containing the log information or empty string if no entries are available.
      */
     public String getLog(String deploymentId, Duration apiTimeout) {
+        logger.info("getLog:{}:{}", deploymentId, apiTimeout);
         Assert.hasText(deploymentId, "id must have text and not null");
         Assert.notNull(apiTimeout, "apiTimeout must not be null");
         StringBuilder stringBuilder = new StringBuilder();
+        ReadRequest request = ReadRequest.builder().sourceId(deploymentId).build();
+        Assert.notNull(request, "request must not be null");
```

Comment:
I don't think this can ever be null since we build it ourselves in the previous line.